src/Umbraco.Web/Models/RelatedLinkBase.cs

```diff
@@ -14,5 +15,7 @@ public abstract class RelatedLinkBase
         public bool IsInternal { get; set; }
         [JsonProperty("type")]
         public RelatedLinkType Type { get; set; }
+        [JsonProperty("content")]
```

Comment:
Good point! I've added it so it can be on the Typed model, I'll remove the JsonProperty attribute.